go.mod

```diff
@@ -4,20 +4,25 @@ go 1.21
 
 require (
 	github.com/PaesslerAG/jsonpath v0.1.1
+	github.com/golang/mock v1.6.0
```

Comment:
We are already using `go.uber.org/mock` which is a maintaned fork of `github.com/golang/mock` please make sure to use it.